react-toggle-switch-component/my-toggle-button/src/ToggleSwitch.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+import './ToggleSwitch.css';
+
+export default function ToggleSwitch({ label }) {
+  const [toggled, setToggle] = useState(true);
+
+  if (toggled) {
+    label = 'ON';
+  } else {
+    label = 'OFF';
+  }
+
+  return (
+    <label>
+      <input
```

Comment:
`input` is an interesting choice. `input` has behavior associated with it that you don't use, so it may be better to just use `div` and give it a className.